company-dcd.el

```diff
@@ -1004,7 +1003,8 @@ If cache was found, use it instead of calling dub."
 (define-key company-dcd-mode-map (kbd "C-c ?") 'company-dcd-show-ddoc-with-buffer)
 (define-key company-dcd-mode-map (kbd "C-c .") 'company-dcd-goto-definition)
 (define-key company-dcd-mode-map (kbd "C-c ,") 'company-dcd-goto-def-pop-marker)
-(define-key company-dcd-mode-map (kbd "C-c s") 'company-dcd-ivy-search-symbol)
+(when (fboundp 'ivy-format-function-arrow)
+  (define-key company-dcd-mode-map (kbd "C-c s") 'company-dcd-ivy-search-symbol))
```

Comment:
If this package is loaded before Ivy, the key will not be bound even if Ivy is actually installed.